qt-models/diveplannermodel.cpp

```diff
@@ -844,6 +844,18 @@ void DivePlannerPointsModel::editStop(int row, divedatapoint newData)
 	 */
 	int old_first_cylid = divepoints[0].cylinderid;
 
+	// Refuse creation of two points with the same time stamp.
+	// Note: "time" is moved in the positive direction to avoid
+	// time becoming zero or, worse, negative.
+	while (std::any_of(divepoints.begin(), divepoints.begin() + row,
+			[t = newData.time] (const divedatapoint &data)
+			{ return data.time == t; }))
+		newData.time += 10;
```

Comment:
Agree with @atdotde - I'd add just 1 second

Reply:
I can't really remember, but I guess I used that since 10 s are used in other places in the same file:
```
                        if (duration <= 0)
                                secs += 10 - duration;
                        p.time = secs;
                        while (++i < divepoints.size())
                                if (divepoints[i].time < divepoints[i - 1].time + 10)
                                        divepoints[i].time = divepoints[i - 1].time + 10;
                        break;
```
and
```
                                int secs = value.toInt() * 60;
                                if (!secs)
                                        secs = 10;
```
and
```
                                if (newtime.seconds == lastrecordedtime.seconds)
                                        newtime.seconds += 10;
```

Of course, this code doesn't ensure that any segment is at least 10 sec. When moving we might fall just one second before or after a different handle and the code is not executed.

I really don't know what the right thing to do is in this case...

Reply:
PS: Fixed the numerous typos in this particular commit message.

Reply:
honestly, I don't know what's right, either... If we consistently used 10 seconds elsewhere... maybe there was a reason?
And of course I forgot to note down the SHA that I reviewed, so now I can't easily tell what you changed since I last looked at this. UGH. Wait, I have a local copy. that should work...
Ahh, no. You rebased (not wrong, please don't misunderstand).
Ok, staring at that diff, I think you changed nothing and just fixed commit messages?